src/patternfly/components/MenuToggle/menu-toggle.hbs

```diff
@@ -15,7 +15,6 @@
     pf-m-expanded
   {{/if}}
   {{#if menu-toggle--modifier}} {{menu-toggle--modifier}}{{/if}}"
-  type="button"
```

Comment:
we'll want to leave this since it should still be a `<button>` by default, and a button is `type="submit"` by default, which we don't want. Though we should wrap it in something like `{{#unless menu-toggle--type}}`, which will unset it if you specify an element other than a `<button>` here. The select toggle does the same

https://github.com/patternfly/patternfly/blob/871aa484334fe6d14cb9f4a479188df97b67e6f6/src/patternfly/components/Select/select-toggle.hbs#L5-L7